charts/opentelemetry-demo/values.yaml

```diff
@@ -500,14 +516,112 @@ opentelemetry-collector:
               allowed_origins:
                 - "http://*"
                 - "https://*"
+      otlp/spanmetrics:
+        protocols:
+          grpc:
+            endpoint: "localhost:65535"
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'otel-collector'
+              scrape_interval: 5s
+              static_configs:
+                - targets: ['0.0.0.0:8888']
```

Comment:
This is not needed, already included in the default config https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/values.yaml#L91-L95